src/collector.js

```diff
@@ -0,0 +1,53 @@
+import { default as Message } from 'message';
+
+// "I have a display case ready and waiting for our newest acquistions!"
+// --Taneleer Tivan
+
+
+export default class Collector {
+
+  constructor() {
+    this.errors = [];
+    this.notices = [];
+    this.warnings = [];
+  }
+
+  getMessageList(type) {
```

Comment:
Ah, fair enough. If that's the only reason we could do:

```js
// Old:
this.getMessageList(type);
// New:
this[`${type}s`]
```
and save adding this method.